subsys/input/Kconfig

```diff
@@ -67,6 +67,20 @@ config INPUT_THREAD_STACK_SIZE
 
 endif # INPUT_MODE_THREAD
 
+config INPUT_EVENT_DUMP
+	bool "Log all input events"
+	depends on LOG
+	help
+	  Dump all input devents using log info messages, has to be enabled
+	  with "input dump on" if INPUT_SHELL is used.
+
+config INPUT_SHELL
+	bool "Input shell"
+	depends on SHELL
```

Comment:
note: all shells typically `default y` right?

Reply:
Yeah debating on that myself, some do, some don't, I have a couple of applications short of flash with a bunch of overrides for shell command that are a bit annoying so I did not do it. Maybe they should be `default y if !SHELL_MINIMAL`?